cli/src/torii/tests.rs

```diff
@@ -30,6 +30,8 @@ use crate::{
     stream::{Sink, Stream},
 };
 
+/* NEEDS TO TALK TO A SUMERAGI
```

Comment:
Attach issue. Anything that's commented out must either be removed or re-enabled.

Reply:
Tests were removed in favor of integration tests.